config/old_reference_configs/cut_kKaHyPar_dissertation.ini

```diff
@@ -45,6 +45,10 @@ i-c-fixed-vertex-acceptance-criterion=fixed_vertex_allowed
 # initial partitioning -> initial partitioning
 i-algo=pool
 i-runs=20
+# initial partitioning -> bin packing
```

Comment:
For example in this config, we shouldn't do bin packing. However, I see that we might run into problems here, because the old code that @kittobi1992 wrote that triggers the repartitioning with tighter balance constraint, is no longer available after merging this. We therefore should create a backup branch that contains the current code, before we merge this PR.

Reply:
I removed the parameters from the old configs and instead adjusted the test cases manually. A backup branch seems reasonable, but currently I dont have the write permissions to create one.